src/main/java/app/models/person/Person.java

```diff
@@ -0,0 +1,43 @@
+package app.models.person;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+import java.util.Objects;
+import java.util.UUID;
+
+@AllArgsConstructor
+@Getter
+public class Person {
+    @Setter
+    UUID id;
+    String name;
+    String surname;
+    String patronymic;
+    String placeOfBirth;
+    Date dateOfBirth;
```

Comment:
`java.util.Date` is an old way of managing dates.
Consisder using new `java.time` package instead.